server/game/Card.js

```diff
@@ -49,7 +49,8 @@ class Card extends EffectSource {
             this.printedKeywords[split[0]] = value;
             this.persistentEffect({
                 location: 'any',
-                effect: AbilityDsl.effects.addKeyword({ [split[0]]: value })
+                effect: AbilityDsl.effects.addKeyword({ [split[0]]: value }),
+                name: split[0] + ':printed'
```

Comment:
was this change on purpose or just to debug?

Reply:
Oops, just to debug. Removed.